bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -238,24 +252,22 @@ export class UIEvents
     @_trigger('rotate:end', e)
 
   _mouse_enter: (e) ->
-    @_bokify_jq(e)
-    @_trigger('move:enter', e)
+    @_bokify_point_event(e)
+    @trigger('move:enter', e)
```

Comment:
I'm fairly sure that everything this PR needs is in ``@_bokify_point_event(e)`` and I don't think switching between ``@trigger`` and ``@_trigger`` should have any effect on how events are processed.